### ISSUE TITLE
crash in analyse_FadingMeasurement()

### ISSUE THREAD

**Issue by mcol** [R-Lum/Luminescence#175]
Using the object object built in `test_analyse_FadingMeasurement.R`, I can crash the function with a simple call:
```R
  # Create artificial object ------------------------------------------------
  l <- list()
  time <- 0
  for(x in runif(3, 120,130)) {
    irr  <- set_RLum(
        "RLum.Data.Curve",
        data = matrix(c(1:x, rep(1,x)), ncol = 2),
        originator = "read_XSYG2R",
        recordType = "irradiation (NA)",
        curveType = "simulated",
        info = list(
          startDate = format(Sys.time() + time, "%Y%m%d%H%M%S"),
          position = 1)
      )

    lum  <- set_RLum(
        "RLum.Data.Curve",
        data = matrix(c(1:40, exp(-c(1:40)/ x * 10)), ncol = 2),
        originator = "read_XSYG2R",
        recordType = "IRSL",
        curveType = "measured",
        info = list(
          startDate = format(Sys.time() + time + x + 30, "%Y%m%d%H%M%S"),
          position = 1)
      )

    time <- time + x + 60
    l <- c(l, irr, lum)
  }

object <- set_RLum("RLum.Analysis", records = l, originator = "read_XSYG2R")
analyse_FadingMeasurement(object, signal.integral = 1:2, background.integral = 3)

# Error in Tx_data[[x]] : subscript out of bounds
```

The crash occurs in this bit of code when `x = 2` as our object has `length(Lx_data) == 2` and `length(Tx_data) == 1`):
https://github.com/R-Lum/Luminescence/blob/759a1142651cc557f9fd06c6b061d1aff244cdf9/R/analyse_FadingMeasurement.R#L395-L398

@RLumSK What's the best approach? Stopping if `length(Lx_data) != length(Tx_data)`, working only up to the longest common length, or something else again?

**Comment by RLumSK** [comment]
The best is here to throw a warning when the lengths differ and use the **shortest** common length.

**Comment by mcol** [comment]
Using the shortest common length, the error moves further down:
```console
Error in data.frame(g_value, TC = tc, G_VALUE_2DAYS = g_value_2days[1],  : 
  arguments imply differing number of rows: 1, 0
```
That occurs because `approx()` fails, so `T_0.5.predict` and `T_0.5.interpolated` are `NULL` and the `T_0.5` data.frame cannot be constructed correctly:
https://github.com/R-Lum/Luminescence/blob/759a1142651cc557f9fd06c6b061d1aff244cdf9/R/analyse_FadingMeasurement.R#L591-L613

Would it be OK in case `approx()` fails to generate directly such a data frame?
```R
T_0.5 <- data.frame(
        T_0.5_INTERPOLATED = NA,
        T_0.5_PREDICTED = NA,
        T_0.5_PREDICTED.LOWER = NA,
        T_0.5_PREDICTED.UPPER = NA
    )
```
The tests we have pass with that change, but I don't think that we actually test the output in such cases.

**Comment by mcol** [comment]
Another problem came up in this snippet of code: 
https://github.com/R-Lum/Luminescence/blob/759a1142651cc557f9fd06c6b061d1aff244cdf9/R/analyse_FadingMeasurement.R#L696-L699
The evaluation of the `expression()` term generated this error:
```console
Error in `plot_RLum.Analysis(object = object, subset = NULL, ...)`: object 'x' not found
```
After a lengthy search, it turns out that using `bquote()` around the expression makes the error go away and the plot title appears correctly.

**Comment by mcol** [comment]
Fixed by #192.
